fiftyone/operators/_types/pipeline.py

```
@@ -63,14 +80,20 @@ class Pipeline:
 
     stages: list[PipelineStage] = dataclasses.field(default_factory=list)
 
+    # ADD A CUSTOM __init__ METHOD TO ACCEPT AND DISCARD UNUSED KWARGS
+    def __init__(self, stages: list[PipelineStage] = None, **kwargs):
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)